examples/CheckInboundIPIsInAFDServiceTagRangeOnConsumptionTier.policy.xml

```diff
@@ -0,0 +1,90 @@
+        <!-- This policy fragment is designed to check the caller's source address comes from inside the AFD Service Tag Range and has a valid AFD-ID header as well as a valid JWT. 
+        It requires a managed identity that has the correct RBAC permissions to call into the service tags api (i.e. has read rights on a subscription)
+        It also needs some named values to be congfigured for it to work.
+        If you wish to use this in the consumption tier (where it makes most sense as you cannot use a service tag to filter this) 
+        you will need to use BYO cache for performance and to avoid throttling on the ARM service tag API. 
+          1. nvfdid - The Front Door ID to check the inbound X-Azure-FDID header for.
+          2. nvoidc - The openid connect URL to use to download the OIDC config for if you want to validate a JWT.
+          3. nvjwtaud - The audience claim to expect in the JWT for the backend.
+          4. nvazsid - A valid Azure subscription ID that the managed identity can read, used in your Service Tag API Call.
+        -->
+        <check-header name="X-Azure-FDID" failed-check-httpcode="401" failed-check-error-message="Not Authorized - route via FrontDoor" ignore-case="true">
+            <value>{{nvfdid}}</value>
+        </check-header>
+        <validate-jwt header-name="Authorization" failed-validation-httpcode="401" failed-validation-error-message="No token provided or token is invalid." require-expiration-time="true" require-signed-tokens="true" clock-skew="300">
+            <openid-config url="{{nvoidc}}" />
+            <audiences>
+                <audience>{{nvjwtaud}}</audience>
+            </audiences>
+        </validate-jwt>
+        <set-header name="Authorization" exists-action="delete" />
+        <cache-lookup-value key="cacheresponse1" variable-name="cacheresponse1" />
+        <choose>
+            <when condition="@(!context.Variables.ContainsKey("cacheresponse1"))">
+                <!-- If the arm context variable doesn’t exist, make an HTTP request to retrieve it.  -->
+                <send-request mode="new" timeout="20" ignore-error="false" response-variable-name="cacheresponse1">
+                    <set-url>https://management.azure.com/subscriptions/{{nvazsid}}/providers/Microsoft.Network/locations/eastus/serviceTags?api-version=2021-03-01</set-url>
+                    <set-method>GET</set-method>
+                    <authentication-managed-identity resource="https://management.azure.com" />
+                </send-request>
+                <set-variable name="cacheresponse1" value="@{
+                    JArray rootvalues = (JArray)((IResponse)context.Variables["cacheresponse1"]).Body.As<JObject>()["values"];
+                    return Newtonsoft.Json.JsonConvert.SerializeObject(rootvalues.Children().Where(e => (string)e["id"] == "AzureFrontDoor.Backend").FirstOrDefault()["properties"]["addressPrefixes"].Select(c => (string)c).ToList());    
+                }" />
```

Comment:
List of strings would be more efficiently stored in a string with a separator, i.e. string1|string2|string3